docs/setup_installation/azure/getting_started.md

```diff
@@ -1,373 +1,193 @@
-</p>
+## Step 1: Connect your Azure account
 
-Go back to the [managed.hopsworks.ai](https://managed.hopsworks.ai) Azure account configuration workflow and proceed to the next step. Copy the CLI command shown:
+[Managed.hopsworks.ai](https://managed.hopsworks.ai) deploys Hopsworks clusters to your Azure account. To enable this, you have to
+create a service principal and a custom role for [managed.hopsworks.ai](https://managed.hopsworks.ai) granting access to your resource group.
 
 <p align="center">
-  <figure>
-    <img style="border: 1px solid #000;width:700px" src="../../../assets/images/setup_installation/managed/azure/connect-azure-9.png" alt="Configure subscription and tenant id">
-    <figcaption>Configure subscription and tenant id</figcaption>
-  </figure>
+  <iframe
+    title="Azure information video"
+    style="width:700px; height: 370px;"
+    src="https://www.youtube.com/embed/Pfx2b3UTt88"
+    frameBorder="0"
+    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+    allowFullScreen
+  >
+  </iframe>
 </p>
 
-Paste the CLI command into your terminal and execute it. Note that you might have multiple entries listed here.
-If so, ensure that you pick the subscription that you want to use.
 
-<p align="center">
-  <figure>
-    <img style="border: 1px solid #000;width:700px" src="../../../assets/images/setup_installation/managed/azure/connect-azure-10.png" alt="Show subscription and tenant id">
-    <figcaption>Show subscription and tenant id</figcaption>
-  </figure>
-</p>
-
-Copy the value of *id* and paste it into the *Subscription id*
-field on [managed.hopsworks.ai](https://managed.hopsworks.ai). Go back to the terminal and copy the value of *tenantId*. Ensure to NOT use the *tenantId* under *managedByTenants*.
-Paste the value into the *Tenant ID* field on [managed.hopsworks.ai](https://managed.hopsworks.ai) and press *Finish*.
+### Step 1.1: Connect your Azure account
 
-Congratulations, you have successfully connected you Azure account to [managed.hopsworks.ai](https://managed.hopsworks.ai).
+In [managed.hopsworks.ai](https://managed.hopsworks.ai/) click on *Connect to Azure* or go to *Settings* and click on *Configure* next to *Azure*. This will direct you to a page with the instructions needed to create the service principal and set up the connection. Follow the instructions.
 
+!!! note 
+    it is possible to limit the permissions that are set up during this phase. For more details see [restrictive-permissions](restrictive_permissions.md).
 <p align="center">
   <figure>
-    <img style="border: 1px solid #000;width:700px" src="../../../assets/images/setup_installation/managed/azure/connect-azure-10.1.png" alt="Store subscription and tenant id">
-    <figcaption>Store subscription and tenant id</figcaption>
+    <img style="border: 1px solid #000;width:700px" src="../../../assets/images/setup_installation/managed/azure/connect-azure-0.png" alt="Cloud account settings">
```

Comment:
can you push the latest screenshot

Reply:
for this one actually i meant and updated version of this https://docs.hopsworks.ai/3.1/assets/images/setup_installation/managed/azure/connect-azure-1.png
similar to what we show for the AWS getting started 
https://docs.hopsworks.ai/3.1/setup_installation/aws/getting_started/#step-1-connecting-your-aws-account